system_baseline/views/v1.py

```diff
@@ -468,17 +557,22 @@ def update_baseline(baseline_id, system_baseline_patch):
         )
         _validate_facts(updated_facts)
         baseline.baseline_facts = updated_facts
-    except FactValidationError as e:
-        raise HTTPError(HTTPStatus.BAD_REQUEST, message=e.message)
+    except FactValidationError as error:
+        message = error.message
+        current_app.logger.audit(message, request=request, success=False)
+        raise HTTPError(HTTPStatus.BAD_REQUEST, message=message)
     except (jsonpatch.JsonPatchException, jsonpointer.JsonPointerException):
-        raise HTTPError(
-            HTTPStatus.BAD_REQUEST, message="unable to apply patch to baseline"
-        )
+        message = "unable to apply patch to baseline"
+        current_app.logger.audit(message, request=request, success=False)
+        raise HTTPError(HTTPStatus.BAD_REQUEST, message=message)
 
     baseline.display_name = system_baseline_patch["display_name"]
 
     baseline.baseline_facts = _sort_baseline_facts(baseline.baseline_facts)
     db.session.add(baseline)
+
+    message = "updating baseline"
```

Comment:
In this PR, all verbs are present (updating, reading, creating) and in this PR: https://github.com/RedHatInsights/historical-system-profiles-backend/pull/118#pullrequestreview-610093806 all verbs are past (read, created, deleted).  It would be nice if this was consistent across our apps, and the logging was taking place before or after the action/query with the matching verb consistently, for ease of adding future work and troubleshooting because the expected output is the same.  I don't have a preference on which and don't know if there's any standard for logging before, after, or both.

Reply:
You are right...

I was thinking if we should log before actual action happens (and thus I used present continuous tense) and then after it succeed (I used past tense).

Reply:
Yeah, either just what fedramp needs or add "before" logs to everything as well and then it's very quick to see where a failure happens as you'll have the before and then lack the after, unless the failed actions captures this already?

Reply:
Failed should be the way to do, most probably :-)

It depends a lot on error handling in each case as I put audit logs to all error places.